docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/msg/lanelet_route.md

```diff
@@ -0,0 +1,14 @@
+# autoware_ad_api_msgs/msg/LaneletRoute
+
+## Definition
+
+```txt
+std_msgs/Header header
+geometry_msgs/Pose start
```

Comment:
Is this optional?

```suggestion
geometry_msgs/Pose[<=1] start
```

Reply:
Fixed https://github.com/autowarefoundation/autoware-documentation/pull/155/commits/211737c245b8ca0f49ab66f84b7693f74afc831d.